### DISCUSSION TITLE
Which audio to backup?

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#558] [Q&A] [answered]
Hello,

I remember having seen in a discussion thread something about backups (I can't find it again) in case something goes very wrong. It was suggested to backup the database, the config file and the "extracted" folder, which I did.

I noticed there are also other folders: Processed, and for  my recent Birdnet-Pi installation, "August-2022" folder and "September-2022" folder.  Should they be backed up too, what are their uses?

Thank you.

**Comment by mcguirepr89** [comment, marked as answer]
Backing up the database, `birdnet.conf`, and the `Extracted` directory will suffice. The dated and `Processed` directories are used during analysis for organizing the files based on their analysis status.

**Reply by blogdemoi** [reply]
Thank you.
